apps/demos/Demos/Chat/Overview/Angular/app/app.component.html

```diff
@@ -0,0 +1,8 @@
+<div class="demo-container">
+    <dx-chat [user]="currentUser" [items]="messages$ | async" [typingUsers]="userChatTypingUsers$ | async"
```

Comment:
Can we move each property to the new line. It makes code mode readable.

Reply:
7d3321bd01507196946a99f35c01832ec2021efd